model/Airbnbs.java

```diff
@@ -0,0 +1,99 @@
+package aireats.model;
+
+import java.math.BigDecimal;
+
+public class Airbnbs {
+    protected String airbnbId;
+    protected String name;
+    protected String city;
+    protected String neighborhood;
+    protected String state;
+    protected BigDecimal latitude;
```

Comment:
type double is enough given the data are of seven digits of accuracy, https://stackoverflow.com/questions/3413448/double-vs-bigdecimal